extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -13,84 +13,77 @@
 @ConfigGroup
 public class OidcCommonConfig {
     /**
-     * The base URL of the OpenID Connect (OIDC) server, for example, `https://host:port/auth`.
-     * OIDC discovery endpoint will be called by default by appending a '.well-known/openid-configuration' path to this URL.
-     * Note if you work with Keycloak OIDC server, make sure the base URL is in the following format:
-     * `https://host:port/realms/{realm}` where `{realm}` has to be replaced by the name of the Keycloak realm.
+     * Optional. The base URL of the OpenID Connect (OIDC) server, for example, `https://host:port/auth`.
```

Comment:
I'm not sure starting with `Optional.` works here as it immediately raises the question, what happens if it is not set.
So my proposal is to drop `Optional.` and instead, after a `<p/>` break, add something like `Do not set it if either public key verification is enabled with a {@link #publicKey} property or a certificate chain verification only is required, see {@link #certificateChain}`.

Something like that

Reply:
Thanks for reviewing this proposal. As discussed, I've dropped `Optional.` from these changes.